### PR TITLE
Most travelled to destinations

### PR DESCRIPTION
**Pull request by captainamiedi**
#### What does this PR do?

- CURD operations to enable users and manager view the most traveled to destinations

#### Description of Task to be completed?

- users and managers can view the most traveled to destinations

#### How should this be manually tested?

- can be tested using postman

#### What are the relevant pivotal tracker stories?

- #167730666

#### Questions:

### CODE REVIEW

File: src/services/travelServices.js [hunk 1]
```diff
@@ -63,3 +66,18 @@ export const showUsertravelsStatus = async (userId) => {
...
+      group : ['destination'],
+      raw: true,
+      order: sequelize.literal('count DESC'),
+      limit: 3
+    });
+  } catch (error) {
+    console.log(error, 'services');
+    throw error;
+  }
+}
```
[the remark this answers is not in the document]
Comment (iakhator) [reply src/services/travelServices.js]: attend to a missing semicolon;

that has been attended to boss. thanks

**Review by iakhator — CHANGES_REQUESTED** [review]

**Review by danoseun — APPROVED** [review]
Nice job @captainamiedi

**Comment by captainamiedi** [comment]
> Nice job @captainamiedi

thanks so much

**Review by chuksjoe — APPROVED** [review]
Great work bro. Well done
Hope you have settled Hound-CI? Also, you can look into the number of commits you have.

**Pull request merged**